### PR TITLE
Azure Storage Persistence saga correlation description

### PR DESCRIPTION
**Pull request by Scooletz**
Connects to https://github.com/Particular/PlatformDevelopment/issues/836

### CODE REVIEW

**Comment by Scooletz** [comment]
Ping @Particular/docs-maintainers

File: nservicebus/azure-storage-persistence/index.md [hunk 1]
```diff
@@ -16,10 +16,21 @@ redirects:
...
+
+To ensure that only one saga can be created for one correlation property value, secondary indexes have been introduced. Their entities are stored in the same table as a saga. When a saga is completed, a secondary index entity is removed as well. It's probable, but highly unlikely, that the saga's completion can leave an orphaned secondary index record. This does not impact the behavior of the persistence as it can detect orphaned records, but may leave a dangling entity in a table with a following `WARN` entry in logs: `Removal of the secondary index entry for the following saga failed: {sagaId}`.
+
+If a migration from 6.2.3 or earlier to a newer version was performed without applying [upgrade guide](/nservicebus/upgrades/asp-saga-deduplication.md), `DuplicatedSagaFoundException` can be observed because of duplicates violating a unique property of a saga. The exception message will include all the information to track down the error for example: `Sagas of type MySaaga with the following identifiers 'id1, id2' are considered duplicates because of the violation of the Unique property CorrelationId.`.
```
Comment (SimonCropp) [inline nservicebus/azure-storage-persistence/index.md]: "upgrade guide" is ambiguous how about `[saga deduplication]`

Comment (SimonCropp) [inline nservicebus/azure-storage-persistence/index.md]: is `'id1, id2' ` really correct? shouldnt it be `'id1', 'id2'`?

Comment (Scooletz) [inline nservicebus/azure-storage-persistence/index.md]: They should be Guids. I'm on the fence either should I pollute this docs with some guids or use this fake ids. What do you think?

Comment (SimonCropp) [inline nservicebus/azure-storage-persistence/index.md]: Perhaps "GUID1"?

Comment (Scooletz) [inline nservicebus/azure-storage-persistence/index.md]: Will do.

**Comment by Scooletz** [comment]
Ping @SimonCropp Comments applied.

**Pull request merged**